deps/rabbit/src/rabbit_amqqueue.erl

```diff
@@ -588,7 +588,7 @@ with(#resource{} = Name, F, E, RetriesLeft) ->
             %% Something bad happened to that queue, we are bailing out
             %% on processing current request.
             E({absent, Q, timeout});
-        {ok, Q} when ?amqqueue_state_is(Q, stopped) andalso RetriesLeft =:= 0 ->
+        {ok, Q = #amqqueue{state = stopped}} andalso RetriesLeft =:= 0 ->
```

Comment:
The helper macros are used for a reason: as records evolve, we cannot be certain that fields won't be renamed or even removed. Please avoid using record matching directly, it has significant negative repercussions for upgrades.